tidb-cloud/migration-job.md

```diff
@@ -0,0 +1,117 @@
+
+- When you delete a cluster, all migration jobs in that cluster are automatically deleted, and the deleted migration jobs are not recoverable.
+
+## Prerequisites
+
+The username you use for the upstream database must have the following privileges:
+
+| Privilege | Scope |
+|:----|:----|
+| `SELECT` | Tables |
+| `RELOAD` | Global |
+| `REPLICATION SLAVE` | Global |
+| `REPLICATION CLIENT` | Global |
+| `PROCESS` | Global |
+
+For example, if you need to migrate the data from `db1` to TiDB Cloud, execute the following `GRANT` statement:
+
+```sql
+GRANT RELOAD, REPLICATION SLAVE, REPLICATION CLIENT, PROCESS ON *.* TO 'your_user'@'your_wildcard_of_host'
+GRANT SELECT ON db1.* TO 'your_user'@'your_wildcard_of_host';
+```
+
+If you also need to migrate the data from other databases to TiDB Cloud, make sure the same privileges are granted to the user of the respective databases.
+
+## Step 1: Go to the **Data Migration** page
+
+1. Log in to the [TiDB Cloud console](https://tidbcloud.com/console/clusters). Navigate to the **Clusters** page for your project.
+
+2. On the **Clusters** page, click the name of your cluster, and click the **Data Migration** tab. The **Data Migration** page is displayed.
+
+3. On the **Data Migration** page, click **Create Migration Job**. The **Create Migration Job** page is displayed.
+
+## Step 2: Configure the source and target connection
+
+On the **Create Migration Job** page, configure the source and target connection.
+
+1. Create a job name. It must consist of numbers and letters with less than 60 characters.
+
+2. Fill in the source connection profile.
+
+   - **Data source**: the data source type. Currently, it supports MySQL, AWS Aurora MySQL, and AWS RDS MySQL.
+   - **Region**: the region of the data source. If the source database is a self-built, the parameter is empty.
+   - **Connectivity method**: the connectivity method of the data source. Currently, it supports Public IP and Private Link.
+   - **Hostname or IP address** (for Public IP): the hostname or IP address of the data source.
+   - **Service Name** (for Private Link): the endpoint service name.
+   - **Port**: the port of the data source.
+   - **Username**: the username of the data source.
+   - **Password**: the password of the data source.
+   - **SSL/TLS** (Only for Public IP): enable or disable the SSL/TLS configuration of the data source. If you enable SSL/TLS, you need to upload the certificate of the data source, including the CA certificate, client certificate, and client key.
+
+3. Fill in the target connection profile.
+
+   - **Username**: enter the username of the target cluster.
+   - **Password**: enter the password of the target cluster.
+
+4. Click **Validate Connection and Next** to validate the information you have entered.
+
+5. You are prompted to add Data Migration service's IP addresses to the IP Access List of your source database and firewall (if any) to allow the Data Migration service to access your source database.
```

Comment:
only `Public IP` will prompt ip allow list. For `private link` user maybe prompted to accept the endpoint request in their account